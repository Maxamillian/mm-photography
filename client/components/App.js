import React, { Component } from 'react';
import { Router, Route } from 'react-router-dom';
import { createBrowserHistory } from 'history';
import Navbar from './Navigation/Navbar';

import Homepage from './Homepage/Homepage';

const history = createBrowserHistory();

class App extends Component {
  constructor(props) {
    super(props);
  }
  render() {
    return (
      <Router history={history}>
        <div>
          Hello world
          <Navbar />
          <Route exact path="/" component={Homepage} />
        </div>
      </Router>
    );
  }
}

export default App;
