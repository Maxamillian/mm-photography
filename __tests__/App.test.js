import React from 'react';
import renderer from 'react-test-renderer';

import App from '../client/components/App';

describe('App', () => {
  it('Renders the app correctly', () => {
    const component = renderer.create(<App />);
    const json = component.toJSON();
    expect(json).toMatchSnapshot();
  });
});
