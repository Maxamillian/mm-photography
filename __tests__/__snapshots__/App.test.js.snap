// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Renders the app correctly 1`] = `
<div>
  Hello world
  <div>
    <a
      className="nav-logo"
      href="/"
      onClick={[Function]}
    >
      MM
    </a>
    <nav
      className="nav primary-nav"
    >
      <ul>
        <li>
          <a
            href="/"
            onClick={[Function]}
          >
            Galleries
          </a>
        </li>
        <li>
          <a
            href="/"
            onClick={[Function]}
          >
            Book now
          </a>
        </li>
        <li>
          <a
            href="/"
            onClick={[Function]}
          >
            Sign in
          </a>
        </li>
      </ul>
    </nav>
  </div>
</div>
`;
